### PR TITLE
Update feedstock

### PR DESCRIPTION
**Pull request by ocefpaf**
`pexpect 4.1.0` with Windows support is out .So I am trying to build this as it is `pexpect` dependency. In this PR:
- the tarball `setup.py` has a date of 1969! That is breaking conda-build. I added an ugly work around until we have a better way to package `flit` packages directly;
- removed the Python 3 constraint as the PyPI classifiers mentioned that py3k is supported;
- removed the Windows constraint, but the module imports `fcntl` and that won't work in Windows. I am not sure how to proceed here as `pexpect` won't build without `ptyprocess`.

### CODE REVIEW

**Comment by takluyver** [comment]
The Windows constraint should remain - Ptyprocess is Unix only (as ptys are a Unix feature). The parts of Pexpect that use it should not be loaded on Windows.

The Python 3 constraint shouldn't have been there to start with - Ptyprocess was Python 3 compatible as soon as it was created.

**Comment by ocefpaf** [comment]
Thanks for the clarification @takluyver. I updated the PR and this should be good to go now.

**Comment by takluyver** [comment]
Do you want to remove appveyor.yml again?

**Comment by ocefpaf** [comment]
> Do you want to remove appveyor.yml again?

I forgot to rerender :flushed:

It should be fixed now.

File: disabled_appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# This file was automatically generated by conda-smithy. To update a component of this
```
Comment (takluyver) [inline disabled_appveyor.yml:1]: It seems a bit odd that it generates this anyway and names it `disabled-` - I assume that's just how conda-smithy works?

Comment (ocefpaf) [inline disabled_appveyor.yml:1]: For `ptyprocess` this looks weird as there won't be a Windows build. @jakirkham decided to keep the AppVeyor yaml configuration in the feedstock as a place holder for those who want to try it in another project. It is harmless to keep that file around.

Note that `conda-smithy` does not really need that file as a new `appveyor.yml` will be generated if the Windows skip is removed.

Comment (takluyver) [inline disabled_appveyor.yml:1]: Fair enough. Looks fine to me, then.

**Comment by takluyver** [comment]
We can merge once Travis has finished.

**Comment by ocefpaf** [comment]
Thanks @takluyver

**Pull request merged**